e2e/terraform/README.md

```diff
@@ -102,6 +102,7 @@ $ source ~/.profile
 ```
 
 ## Creating the Config For Cluster
```

Comment:
Configuring Cluster or is it Creating the Configuration File for Cluster?

Reply:
Creating the Configuration File for Cluster